scripts/sct_dmri_moco.py

```diff
@@ -369,36 +372,30 @@ def dmri_moco(param):
 
     # DWI groups
     file_dwi_mean = []
+    tqdm_bar = tqdm(total=nb_groups, unit='iter', unit_scale=False, desc="Merge within groups", ascii=True, ncols=80)
     for iGroup in range(nb_groups):
```

Comment:
You could have used a tqdm iterator object instead of the range() here (avoids to `update()` and `close()`)

Reply:
i was not aware of that-- i'll look into it

Reply:
fixed in c68600ddc6cc801ebe7ebf52aa7373f1d651050b